readthedocs/basic/signing-in.rst

```diff
@@ -127,15 +127,36 @@ with
 
 .. code-block:: python
 
-    TelegramClient('anon', api_id, api_hash, proxy=(socks.SOCKS5, '127.0.0.1', 4444))
+    TelegramClient('anon', api_id, api_hash, proxy=("socks5", '127.0.0.1', 4444))
 
 (of course, replacing the IP and port with the IP and port of the proxy).
 
 The ``proxy=`` argument should be a tuple, a list or a dict,
-consisting of parameters described `in PySocks usage`__.
+consisting of parameters described `in aiosocks usage`__.
 
-.. __: https://github.com/Anorov/PySocks#installation
-.. __: https://github.com/Anorov/PySocks#usage-1
+Example:
+
+* .. code-block:: python
+
+    proxy = ('socks5', '1.1.1.1', 5555, 'foo', 'bar', True)
```

Comment:
I'm not sure if I want to commit to a tuple format for proxies. They're hard to read and can't be easily updated to a different format in the future. Maybe the new-style would be the `dict` below (no `tuple` or `list`), and for support with PySocks, their `tuple` style may be used.

Reply:
@Lonami 
Latest pull commit makes it possible to use `PySocks` `dict` format instead of `tuple`.